doc/source/whatsnew/v1.3.0.rst

```diff
@@ -145,6 +145,7 @@ Other enhancements
 - :class:`RangeIndex` can now be constructed by passing a ``range`` object directly e.g. ``pd.RangeIndex(range(3))`` (:issue:`12067`)
 - :meth:`round` being enabled for the nullable integer and floating dtypes (:issue:`38844`)
 - :meth:`pandas.read_csv` and :meth:`pandas.read_json` expose the argument ``encoding_errors`` to control how encoding errors are handled (:issue:`39450`)
+- :meth:`.Styler.clear` now clears :attr:`Styler.hidden_index`, :attr:`Styler.hidden_columns` and :attr:`Styler.tooltips` as well (:issue:`40484`)
```

Comment:
remove `tooltips` since it already does this.

Reply:
is this user visible at all?

Reply:
this is a user visible change only if: 
- the user has a styler which has a hidden index, or hidden columns applied, or both,
- then uses the `clear` method, 
- then re-renders the Styler

prior to this PR the hidden items would still be hidden, after this PR the styler resets so nothing is hidden.